ansible/roles/runner/tasks/setup.yml

```diff
@@ -39,9 +45,29 @@
     - python3-tqdm
     - python3-pytz
     - python3-jinja2
+    - python-firewall
+    - python3-firewall
   notify:
     - restart_nfs
 
+- name: enable firewalld
+  service:
+    name: firewalld
+    enabled: true
+    state: started
+
+- name: configure firewalld
+  firewalld:
+    service : "{{ item }}"
+    permanent: true
+    state: enabled
+  with_items:
+    - nfs
+    - ipp
+    - rpc-bind
+    - mountd
+  notify: firewalld complete reload
```

Comment:
This was addressed by another PR right? so it should not be on this on I guess

Reply:
While another one is not merged I need to have it somewhere.

Will remove this part later.